src/Rules/Arrays/InvalidKeyInArrayItemRule.php

```diff
@@ -22,6 +23,7 @@ final class InvalidKeyInArrayItemRule implements Rule
 
 	public function __construct(
 		private RuleLevelHelper $ruleLevelHelper,
+		private PhpVersion $phpVersion,
```

Comment:
More modern approach is to get PhpVersions from Scope (a separate class).

Reply:
Does it means that all the method from PhpVersion should be deprecated/moved into PhpVersions ?

Reply:
Not yet